_topic_maps/_topic_map_rosa_hcp.yml

```diff
@@ -474,6 +474,55 @@ Topics:
   - Name: Important changes to OpenShift Jenkins images
     File: important-changes-to-openshift-jenkins-images
 ---
+Name: Images
+Dir: openshift_images
+Distros: openshift-rosa-hcp
+Topics:
+- Name: Overview of images
+  File: index
+# replaced Configuring the Cluster Samples Operator name, cannot configure the operator
+- Name: Overview of the Cluster Samples Operator
+  File: configuring-samples-operator
+  Distros: openshift-rosa-hcp
+- Name: Using the Cluster Samples Operator with an alternate registry
+  File: samples-operator-alt-registry
+  Distros: openshift-rosa-hcp
+- Name: Creating images
+  File: create-images
+- Name: Managing images
+  Dir: managing_images
+  Topics:
+  - Name: Managing images overview
+    File: managing-images-overview
+  - Name: Tagging images
+    File: tagging-images
+  - Name: Image pull policy
+    File: image-pull-policy
+  - Name: Using image pull secrets
+    File: using-image-pull-secrets
+- Name: Managing image streams
+  File: image-streams-manage
+  Distros: openshift-rosa-hcp
+- Name: Using image streams with Kubernetes resources
+  File: using-imagestreams-with-kube-resources
+  Distros: openshift-rosa-hcp
+- Name: Triggering updates on image stream changes
+  File: triggering-updates-on-imagestream-changes
+  Distros: openshift-rosa-hcp
+- Name: Image configuration resources (HCP)
+  File: image-configuration-hcp
+  Distros: openshift-rosa-hcp
+- Name: Using images
+  Dir: using_images
+  Distros: openshift-rosa-hcp
+  Topics:
+  - Name: Using images overview
+    File: using-images-overview
+  - Name: Source-to-image
+    File: using-s21-images
+  - Name: Customizing source-to-image images
+    File: customizing-s2i-images   
```

Comment:
This is very silly, but there are a few trailing spaces after `File: customizing-s2i-images`. Removing those allowed it to build locally, so it's worth try

Reply:
Thank you!